text_generation/causal_lm/cpp/continuous_batching/library/src/sampler.hpp

```diff
@@ -295,42 +295,51 @@ class Sampler {
         return out_token;
     }
 
-    int64_t _multinomial_sample(ov::Tensor logits, float temperature, float top_p, size_t top_k) {
+    std::vector<int64_t> _multinomial_sample(ov::Tensor logits, float temperature, float top_p, size_t top_k, size_t n) {
```

Comment:
```suggestion
    std::vector<int64_t> _multinomial_sample(ov::Tensor logits, float temperature, float top_p, size_t top_k, size_t num_tokens_per_sequence) {
```
can we name it more specifically here? because depending on iteration, it may have different value and it's different from original `n` (num return sequences)